cclient/serialization/json/json_serializer.c

```diff
@@ -40,6 +40,52 @@ retcode_t utarray_to_json_array(UT_array* ut, cJSON* json_root,
   return RC_OK;
 }
 
+retcode_t json_get_size(const cJSON* json_obj, const char* obj_name,
+                        size_t* num) {
+  cJSON* json_value = cJSON_GetObjectItemCaseSensitive(json_obj, obj_name);
+  if (json_value == NULL) return RC_CCLIENT_JSON_KEY;
+
+  if (cJSON_IsNumber(json_value)) {
```

Comment:
Can it be negative? would you return error in that case?

Reply:
it's a boolean and return RC_CCLIENT_JSON_PARSE if the value is not a number.

Reply:
Yes, i was referring to: `(size_t)json_value->valuedouble`
could that line be casting negative number to `size_t`?

Reply:
could the value be negative in the response? ex. memory size, time, index of a milestone...